indico/modules/rb_new/client/js/components/Room.jsx

```diff
@@ -21,14 +21,12 @@ import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
 import {Card, Icon, Label, Popup, Button} from 'semantic-ui-react';
-import roomsSpriteURL from 'indico-url:rooms_new.sprite';
 
 import {Translate} from 'indico/react/i18n';
 import {Slot} from 'indico/react/util';
 import {TooltipIfTruncated} from 'indico/react/components';
 import SpriteImage from './SpriteImage';
-import DimmableImage from './Dimmer.jsx';
-import {selectors as configSelectors} from '../common/config';
+import DimmableImage from './DimmableImage.jsx';
```

Comment:
no need for the `.jsx` here

Reply:
Amazing how I changed the name of the component and didn't notice the extension 🤣

Reply:
shh, you could blame your editor for doing it automatically when renaming the file! ;)